paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -255,6 +256,13 @@ public class PaparazziPlugin @Inject constructor(
 
         test.outputs.dir(reportOutputDir).withPropertyName("paparazzi.report.dir")
 
+        if (test.javaVersion.isCompatibleWith(JavaVersion.VERSION_21)) {
+          // Enabled dynamic agent loading (ByteBuddy) for JDK 21+ https://openjdk.org/jeps/451
+          test.jvmArgs = (test.jvmArgs?.toMutableList() ?: mutableListOf()).apply {
+            add("-XX:+EnableDynamicAgentLoading")
+          }
```

Comment:
consider:
```suggestion
          test.jvmArgs = test.jvmArgs.orEmpty() + "-XX:+EnableDynamicAgentLoading"
```
does the result **have to** be mutable?